### ISSUE TITLE
 more options for C-x and C-c behavior

### ISSUE THREAD

**Issue by jasonperry** [jyp/boon#138]
Thanks very much for boon, after much research it's the modal package that best fits my needs, especially with the C-x and C-c bindings.

My feature request is to provide more options for how the C-x and C-c maps work. Specifically:
- Have a setting or alternate functions that provide the opposite behavior for whether Ctrl is held for subsequent keys. This is more important for c-god, but if it could be implemented for boon-x-map as well, it would mean fewer custom keybinds would need to be made.  
- Optionally, have a key to toggle whether Ctrl is held for subsequent keys, like Space does in god-mode.

**Comment by YassinRian** [comment]
agree ...Boon is from my perspective very underrated. It strikes the right balance between keeps Emacs keybindings en having some nice modal functions

**Comment by jyp** [comment]
So, to clarify, you'd like `c a` to do `C-c a` and not `C-c C-a`, for instance?/

**Comment by jasonperry** [comment]
> So, to clarify, you'd like `c a` to do `C-c a` and not `C-c C-a`, for instance?/

Yes, that's right, as an option. I know there are lots of existing bindings for both cases, so maybe there should be two separate functions.

**Comment by YassinRian** [comment]
Make use of Hydra's ...they will give you extra super powers:) ...you can attach all kind of hydra's to the menu chars w / e / g.
